addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -189,18 +189,20 @@
                         </group>
                         <notebook>
                             <page string="Mail Body" name="mail_body">
-                                <div class="mt-n2">
-                                    <field name="body_html" class="o_mail_body oe_read_only" widget="html"
+                                <field name="is_body_empty" invisible="1"/>
+                                <div class="mt-n2 oe_read_only" attrs="{'invisible': [('is_body_empty', '=', True)]}">
+                                    <field name="body_html" class="o_mail_body" widget="html"
                                         options="{'cssReadonly': 'mass_mailing.iframe_css_assets_readonly'}"/>
-                                    <field name="body_arch" class="o_mail_body oe_edit_only" widget="mass_mailing_html"
+                                </div>
+                                <div class="mt-n2 oe_edit_only">
+                                    <field name="body_arch" class="o_mail_body" widget="mass_mailing_html"
                                         options="{
                                             'snippets': 'mass_mailing.email_designer_snippets',
                                             'cssEdit': 'mass_mailing.iframe_css_assets_edit',
                                             'inline-field': 'body_html'
                                     }" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
                                 </div>
-                                <field name="is_body_empty" invisible="1"/>
-                                <div class="o_view_nocontent oe_read_only" attrs="{'invisible': ['|', ('is_body_empty', '=', False), ('state', 'in', ('sending', 'done'))]}">
+                                <div class="o_view_nocontent_block oe_read_only" attrs="{'invisible': ['|', ('is_body_empty', '=', False), ('state', 'in', ('sending', 'done'))]}">
```

Comment:
How about using `"o_view_nocontent oe_read_only position-static"` ?
I tested in developer mode and it seems to work

I always prefer not introducing new classes into the web framework if we can avoid it..

Or even better, since absolute-positioned element are always positioned relatively to their closest parent that is positioned "relative" or "absolute", adding:
`class="position-relative"`
to:
`<page string="Mail Body" name="mail_body">`

Could do the trick without changing anything else, making this commit a one-line fix.
I have not tried that though, it requires a bit more than just fooling around with the developer mode on a runbot.

Reply:
I didn't think of the `position-static`. I will remove the custom class `o_view_nocontent_block` and use that instead. 

I think that it's better to remove the absolute positionning.

With a position `absolute`, we have to (1) know the dimensions of the action helper and (2) fix a min-width and a min-height for the parent container so that the action helper will not overflow. If the dimensions of the container or the action helper change, we would have the same issue.

With a position `static`, the element will be in the flow of the document. The container will automatically have the width and the height of the action helper which is really convenient.